controllers/v1/events/events.controller.js

```diff
@@ -25,7 +25,6 @@ const get = async (request, response) => {
 
   try {
     const events = await eventsService.doList(eventParameters)
```

Comment:
En todo lugar donde podamos hacer destructuring hagamoslo
```suggestion
    const { responseCode,  data, message } = await eventsService.doList(eventParameters)
```